CHANGELOG.rst

```diff
@@ -1,34 +1,44 @@
 Release Notes
 =============
 
-Changes are ordered reverse-chronologically.
+List of the most important changes for each release.
 
 0.11.0
 ------
 
- - Upgrades to exam and lesson creation, including search functionality and auto-save
+Added
+~~~~~
+
  - Support for EPUB-format electronic books
+ - Upgrades to exam and lesson creation, including search functionality and auto-save
  - New error handling and reporting functionality
- - Added a new setting for enabling or disabling guest access
- - Auto-refresh in coach reports
- - Import from custom network locations
+ - Channel import from custom network locations
+ - Setting for enabling or disabling guest access
+ - Basic commands to help with GDPR compliance
+ - Privacy information to help users and admins understand how their data is stored
+
+Changed or fixed
+~~~~~~~~~~~~~~~~
+
+ - Improvements to rendering of some pages on smaller screens
  - Improvements to search behavior in filtering and handling of large result sets
- - Added some basic commands to help with GDPR compliance
- - Added privacy information to help users and admins understand how their data is stored
- - Improved consistency of language across the application, and renamed "Superuser" to "Super admin"
- - Many fixes to internationalization
- - Consistent font rendering across all languages
- - Allow usernames in non-latin alphabets
  - Improvements to the setup wizard based on user feedback and testing
  - Improvements to user management, particularly for admins and super admins
+ - Fix: Allow usernames in non-latin alphabets
+ - Fix: Drive listing and space availability reporting
+ - Auto-refresh in coach reports
  - Added more validation to help with log-in
- - Improvements in drive listing and space availability reporting
- - Improved rendering of some pages on smaller screens
  - Security: upgraded Python cryptography and pyopenssl libraries for CVE-2018-10903
- - Many additional bug fixes, under-the-hood updates, and performance improvements
+
+Translations
+~~~~~~~~~~~~
+
  - Languages: [ coming soon... ]
+ - Improved consistency of language across the application, and renamed "Superuser" to "Super admin"
+ - Many fixes to internationalization
```

Comment:
Many fixes to translation and localization?

Reply:
Alright, corrected... am a bit confused about the distinction between internationalization, translation and localization at this point.. but please have a look at the current state :)